.drone.yml

```diff
@@ -4,7 +4,9 @@ workspace:
 
 pipeline:
   build:
-    image: golang:1.10
+    image: golang:1.14
```

Comment:
Does this need to match the version in go.mod?

Reply:
oh sorry, that's my local Golang. will change it, thought of keeping it stable version of golang and not latest.